themes/bootprint3/less/icons.less

```diff
@@ -1,151 +1,13 @@
-i.fa-rss { background-image: url("@{bp3-icon-path}/feed.png"); &:extend(.bp-icon); }
-i.fa-search { background-image: url("@{bp3-icon-path}/magnifier.png"); &:extend(.bp-icon); }
-i.fa-search-plus { background-image: url("@{bp3-icon-path}/magnifier_zoom_in.png"); &:extend(.bp-icon); }
-i.fa-sensorimage { background-image: url("@{bp3-icon-path}/photo.png"); &:extend(.bp-icon); }
-i.fa-serial { background-image: url("@{bp3-icon-path}/page_white_stack.png"); &:extend(.bp-icon); }
-i.fa-shopping-cart { background-image: url("@{bp3-icon-path}/cart.png"); &:extend(.bp-icon); }
-i.fa-sign-in { background-image: url("@{bp3-icon-path}/door_in.png"); &:extend(.bp-icon); }
-i.fa-sign-out { background-image: url("@{bp3-icon-path}/door_out.png"); &:extend(.bp-icon); }
-i.fa-slide { background-image: url("@{bp3-icon-path}/film.png"); &:extend(.bp-icon); }
-i.fa-software { background-image: url("@{bp3-icon-path}/drive_cd.png"); &:extend(.bp-icon); }
-i.fa-soundcassette { background-image: url("@{bp3-icon-path}/sound.png"); &:extend(.bp-icon); }
-i.fa-sounddisc { background-image: url("@{bp3-icon-path}/cd.png"); &:extend(.bp-icon); }
-i.fa-soundrecording { background-image: url("@{bp3-icon-path}/sound.png"); &:extend(.bp-icon); }
-i.fa-spinner { background-image: url("@{bp3-icon-path}/ajax_loading.gif"); &:extend(.bp-icon); }
-i.fa-status-unknown { background-image: url("@{bp3-icon-path}/bullet_orange.png"); &:extend(.bp-icon); }
-i.fa-suitcase { background-image: url("@{bp3-icon-path}/bookbag.png"); &:extend(.bp-icon); }
-i.fa-tapecartridge { background-image: url("@{bp3-icon-path}/drive.png"); &:extend(.bp-icon); }
-i.fa-tapecassette { background-image: url("@{bp3-icon-path}/drive.png"); &:extend(.bp-icon); }
-i.fa-tapereel { background-image: url("@{bp3-icon-path}/film.png"); &:extend(.bp-icon); }
-i.fa-transparency { background-image: url("@{bp3-icon-path}/film.png"); &:extend(.bp-icon); }
-i.fa-tree { background-image: url("@{bp3-icon-path}/treeCurrent.png"); &:extend(.bp-icon); }
-i.fa-tree-muted { background-image: url("@{bp3-icon-path}/treeMuted.png"); &:extend(.bp-icon); }
-i.fa-unknown { background-image: url("@{bp3-icon-path}/page_white.png"); &:extend(.bp-icon); }
-i.fa-usd { background-image: url("@{bp3-icon-path}/money_dollar.png"); &:extend(.bp-icon); }
-i.fa-user { background-image: url("@{bp3-icon-path}/user.png"); &:extend(.bp-icon); }
-i.fa-video{background-image: url("@{bp3-icon-path}/television.png"); &:extend(.bp-icon); }
-i.fa-videocartridge { background-image: url("@{bp3-icon-path}/television.png"); &:extend(.bp-icon); }
-i.fa-videocassette{background-image: url("@{bp3-icon-path}/television.png"); &:extend(.bp-icon); }
-i.fa-videodisc{background-image: url("@{bp3-icon-path}/cd.png"); &:extend(.bp-icon); }
-i.fa-videoreel{background-image: url("@{bp3-icon-path}/film.png"); &:extend(.bp-icon); }
-i.fa-visual { background-image: url("@{bp3-icon-path}/view_visual.png"); &:extend(.bp-icon); }
-
+/*
```

Comment:
What's the status of this right-to-left stuff?

Reply:
The above TODO is for this commented out code. There is currently no official way to do RTL but there are ways to support it with fancy classes like `$this->icon('icon', 'rtl:icon-rtl')` or something. Not for 8.0.

Reply:
I think we should hold off on RTL icons for now